### PR TITLE
Send all data to the client at startup

### PR DESCRIPTION
**Pull request by ostcar**

### CODE REVIEW

File: openslides/core/apps.py [hunk 1]
```diff
@@ -55,3 +55,10 @@ def ready(self):
         signals.m2m_changed.connect(
             index_add_instance,
             dispatch_uid='m2m_index_add_instance')
+
+    def get_startup_elements(self):
+        from .config import config
+        from ..utils.collection import Collection
+        for model in ('Projector', 'ChatMessage', 'Tag', 'ProjectorMessage', 'Countdown'):  #TODO Config
```
Comment (normanjaeckel) [inline openslides/core/apps.py]: Remove config TODO.

File: openslides/utils/autoupdate.py [hunk 2]
```diff
@@ -19,12 +20,33 @@ def ws_add_site(message):
     Adds the websocket connection to a group specific to the connecting user.
 
     The group with the name 'user-None' stands for all anonymous users.
+
+    Send all "starup-data" through the connection.
```
Comment (normanjaeckel) [inline openslides/utils/autoupdate.py]: Typo.

File: openslides/utils/autoupdate.py [hunk 3]
```diff
@@ -19,12 +20,33 @@ def ws_add_site(message):
...
+            # Get the method get_startup_elements() from an app.
+            # This method has to return an iterable of Collection objects.
+            get_startup_elements = app.get_startup_elements
+        except AttributeError:
+            # Skip apps that do not implement get_startup_elements
+            continue
+        for collection in get_startup_elements():
+            output.extend(collection.as_autoupdate_for_user(message.user))
+
+    # Send all data. If there is no data, then onyl accept the connection
```
Comment (normanjaeckel) [inline openslides/utils/autoupdate.py]: Typo `only`

**Comment by emanuelschuetze** [comment]
see #2863

**Pull request closed without merging**